assets/service_now_notifications_handler/index.py

```diff
@@ -783,61 +990,98 @@ def handler(event: Dict[str, Any], context: LambdaContext) -> Dict[str, Any]:
         API Gateway compatible response
     """
     try:
-        # Log incoming event
+        # Log incoming event with more details for debugging
         logger.info(f"Received event: {json.dumps(event)}")
```

Comment:
question(blocking): How likely is it that there is sensitive information in the body of the event? If there is any likelihood, how are we handling that situation?